app/presenters/licence_details_presenter.rb

```diff
@@ -18,7 +18,20 @@ def local_authority_specific?
   def licence_authority_specific?
     !licence_details["location_specific"]
   end
-
+  
+  def any_action_uses_licensify?
+    authority["actions"] && authority["actions"].keys.uniq.any?{|chosen_action| uses_licensify(chosen_action) == true }
+  end
+  
+  def uses_licensify(chosen_action)
+    return false unless chosen_action
+    authority["actions"] && authority["actions"][chosen_action][0]['uses_licensify']
```

Comment:
Do we need to check _all_ the possible links for a given action, rather than the first one?  I'm not 100% sure of the structure of the response but it sounds like: there's 1 authority, it can have zero or more actions, and each of those can have one or more links (not sure about these names but they'll do for now).  In the view when rendering the info about each link we check specifically the `uses_licensify` details of each link before rendering it, but we also guard the navigation that would take us to the page where we render those links by using this method and it only checks the first link for each action.

Reply:
Checking all links